structopt-derive/src/lib.rs

```diff
@@ -900,14 +901,5 @@ fn impl_structopt(ast: &DeriveInput) -> quote::Tokens {
         _ => panic!("structopt only supports non-tuple structs and enums")
     };
 
-    let dummy_const = Ident::new(format!("_IMPL_STRUCTOPT_FOR_{}", struct_name));
-    quote! {
-        #[allow(non_upper_case_globals)]
-        #[allow(unused_attributes, unused_imports, unused_variables)]
-        const #dummy_const: () = {
-            extern crate structopt as _structopt;
-            use structopt::StructOpt;
```

Comment:
There was a bug here, making the trick quite useless: this line require a module `structopt` at the root that we didn't created in the dummy const. The line should be `use _structopt::StructOpt;` (moreover, this import was useless...).

As using structop-derive without using structopt is quite useless, just remove that as that will fix #45